### DISCUSSION TITLE
SCTransform across multiple experimental conditions or samples

### DISCUSSION THREAD

**Discussion by errcricket** [satijalab/seurat#7093] [Q&A]
Greetings.

I have a clarifying question from [https://github.com/satijalab/seurat/issues/6054](https://github.com/satijalab/seurat/issues/6054) (and the Seurat [vignettes](https://satijalab.org/seurat/articles/sctransform_v2_vignette.html) that generally show two samples (each with a different experimental condition)).

_The goal of SCTransform is to perform normalization within an experiment by learning the model of technical noise (within the experiment). If you have multiple experiments, we recommend running SCTransform on each experiment individually._

I have 12 samples across 4 experimental conditions (3 biological replicates per condition); in the last sentence of the quoted text, does `each experiment` mean `each experimental condition` or `each sample` as it pertains to my data? I thought it should have been on samples, but this [post](https://github.com/satijalab/seurat/discussions/5702) has me thinking it should be on experimental condition.

Thank you in advance for your response.

**Comment by jayypaul** [comment]
I'm also wondering this.

Thanks,

Jordan
